taca/analysis/analysis.py

```diff
@@ -40,11 +40,11 @@ def _run_type(run):
         #this information about the run type (with HiSeq2.5 applicationaName does not work anymore, but as for a long time we will have instruments not updated I need to find out somehting that works
         try:
             #Works for recent control software
-            runtype=rp.data['RunParameters']["Setup"].get("Flowcell")
+            runtype=rp.data['RunParameters']["Setup"]["Flowcell"]
         except KeyError:
             #use this as second resource but print a warning in the logs
             logger.warn("Parsing runParameters to fecth instrument type, not found Flowcell information in it. Using ApplicaiotnName")
-            runtype=rp.data.get("ApplicationName")
```

Comment:
I think `obj['RunParameters']["Setup"]` is created by `flowcell_parser` and I assume this always works (so these keys will be present when it is needed). But if it is not wouldn't it fail ? and is that what should happen ?